nirc_ehr/resources/etls/project.xml

```diff
@@ -7,7 +7,7 @@
             <description>Copy to project target</description>
             <source schemaName="dbo" queryName="q_project"/>
             <destination schemaName="ehr" queryName="project"
-                         bulkLoad="true" targetOption="merge">
```

Comment:
This may have come up in conversation earlier, but why is this changing from merge? I would expect the default of "append" to cause headaches, either with duplicate rows or constraint violations when it's rerun.

Reply:
having "merge" as target option is storing only the last row of the projects and protocols. I remember having the conversation earlier for a similar scenario but forgot what was done to resolve the issue.

Reply:
That's a sign that all rows are using the same PK value according to the target table. I think we need to figure this out or it'll cause problems going forward.

Reply:
done